README.md

```diff
@@ -18,12 +18,9 @@ To add a set to your config, use `Rector\Doctrine\Set\DoctrineSetList` class and
 
 ```php
 use Rector\Config\RectorConfig;
-use Rector\Doctrine\Set\DoctrineSetList;
 
 return RectorConfig::configure()
-    ->withSets([
-        DoctrineSetList::DOCTRINE_CODE_QUALITY,
-    ]);
+    ->withAttributesSets(doctrine: true);
```

Comment:
should be:

```suggestion
    ->withPreparedSets(doctrineCodeQuality: true);
```

Reply:
Sorry i did it a bit too hastily. I propose some changes in the commit [4420a1d](https://github.com/rectorphp/rector-doctrine/pull/328/commits/4420a1d3de94e4093b237887ec576fc3bb08d5a7).
However the documentation for [doctrine rules](https://github.com/rectorphp/rector-doctrine/blob/main/docs/rector_rules_overview.md) also needs an updates.